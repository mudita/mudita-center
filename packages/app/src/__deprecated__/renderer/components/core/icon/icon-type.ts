/**
 * Copyright (c) Mudita sp. z o.o. All rights reserved.
 * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
 */

export enum IconType {
  Arrow,
  ArrowLongLeft,
  ArrowLongRight,
  AttachContact,
  BackupFolder,
  Battery,
  BorderCheckIcon,
  Calendar,
  CalendarIcon,
  ChargingBattery,
  Check,
  CheckCircle,
  CheckIndeterminate,
  Cloud,
  Close,
  CloseWhite,
  Delete,
  FilesManager,
  HighBattery,
  HighRange,
  LowBattery,
  LowRange,
  Magnifier,
  MenuFilesManager,
  MenuHelp,
  MenuContacts,
  MenuMeditation,
  MenuMusic,
  MenuNews,
  MenuOverview,
  MenuPhone,
  MenuSettings,
  MenuTethering,
  MenuTools,
  MenuRange,
  MenuBattery,
  Message,
  ExternalLink,
  MuditaLogo,
  MuditaLogoWithText,
  MuditaLogoVertical,
  MediumBattery,
  MediumRange,
  NoBattery,
  NoRange,
  Refresh,
  Send,
  Sim,
  Support,
  Template,
  Calls,
  Connection,
  Contact,
  NewContact,
  Dial,
  Notes,
  Notifications,
  Playlist,
  PlusSign,
  Reload,
  Upload,
  UploadDark,
  VeryHighBattery,
  VeryHighRange,
  VeryLowBattery,
  VeryLowRange,
  VoiceRecorder,
  Info,
  Download,
  DownloadWhite,
  Pure,
  Fail,
  FailRed,
  More,
  Blocked,
  Forward,
  Edit,
  Favourites,
  Ice,
  ArrowDown,
  ArrowDownWhite,
  DeleteBig,
  Attachment,
  Remove,
  SendButton,
  Backup,
  Tooltip,
  TetheringStatus,
  Apple,
  Google,
  SynchronizeContacts,
  Templates,
  MissedCall,
  ConferenceCall,
  IncomingCall,
  OutgoingCall,
  MusicGrey,
  VoiceRecorderGrey,
  FullBattery,
  VeryHighRangeWithRoaming,
  HighRangeWithRoaming,
  MediumRangeWithRoaming,
  LowRangeWithRoaming,
  VeryLowRangeWithRoaming,
  ThinCheck,
  LongArrow,
  MarkAsRead,
  RecoveryBackup,
  RebootOs,
  FactoryReset,
  MuditaFlower,
  MuditaDarkLogo,
  ContactGoogle,
  Manage,
  MuditaLogoBg,
  Outlook,
  ContactFilled,
  Visible,
  Hidden,
  ThinFail,
  Warning,
  MarkAsUnread,
}
