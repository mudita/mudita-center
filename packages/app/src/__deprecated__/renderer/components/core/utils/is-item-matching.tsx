/**
 * Copyright (c) Mudita sp. z o.o. All rights reserved.
 * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
 */

export type IsItemMatching = (item: any, searchString: string) => boolean
export const isItemValueMatching: IsItemMatching = (
  itemValue: string,
  search
) => itemValue.toLowerCase().includes(search.toLowerCase())
