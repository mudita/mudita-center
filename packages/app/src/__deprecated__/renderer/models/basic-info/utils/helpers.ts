/**
 * Copyright (c) Mudita sp. z o.o. All rights reserved.
 * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
 */

import { SimCard } from "App/__deprecated__/renderer/models/basic-info/basic-info.typings"

export const getActiveNetworkFromSim = (simCards: SimCard[] = []) => {
  const activeSimCard = simCards.filter((simCard) => simCard.active)
  if (activeSimCard.length === 0) {
    return "No connection"
  }
  return activeSimCard[0].network
}
export const getActiveNetworkLevelFromSim = (simCards: SimCard[] = []) => {
  const activeSimCard = simCards.filter((simCard) => simCard.active)
  if (activeSimCard.length === 0) {
    return "No connection"
  }
  return activeSimCard[0].networkLevel
}
