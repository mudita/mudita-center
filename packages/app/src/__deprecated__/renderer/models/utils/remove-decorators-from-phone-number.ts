/**
 * Copyright (c) Mudita sp. z o.o. All rights reserved.
 * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
 */

export const removeDecoratorsFromPhoneNumber = (phoneNumber = ""): string => {
  const rawPhoneNumber = phoneNumber.replace(/[^\d]/g, "")
  if (rawPhoneNumber === "") {
    return phoneNumber.replace(/\s/g, "").toLowerCase()
  } else {
    return rawPhoneNumber
  }
}
