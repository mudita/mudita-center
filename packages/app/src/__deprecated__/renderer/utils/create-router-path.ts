/**
 * Copyright (c) Mudita sp. z o.o. All rights reserved.
 * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
 */

import { URL_MAIN, URL_TABS } from "App/__deprecated__/renderer/constants/urls"

//if params contains sensitive data, they should be added to scrubFields array in logger.ts
const createRouterPath = (
  link:
    | typeof URL_MAIN[keyof typeof URL_MAIN]
    | typeof URL_TABS[keyof typeof URL_TABS],
  params: Record<string, string>
): string => {
  return `${link}?${new URLSearchParams(params).toString()}`
}

export default createRouterPath
