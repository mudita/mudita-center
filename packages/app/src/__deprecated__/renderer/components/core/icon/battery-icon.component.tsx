/**
 * Copyright (c) Mudita sp. z o.o. All rights reserved.
 * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
 */

import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
import * as React from "react"
import Icon, {
  Props as IconProps,
} from "App/__deprecated__/renderer/components/core/icon/icon.component"
import { IconType } from "App/__deprecated__/renderer/components/core/icon/icon-type"

export interface BatteryIconProps {
  level: number
  charging?: boolean
}

const getInteractiveBatteryIcon = (
  batteryLevel: number,
  charging: boolean,
  size: number,
  rest: Partial<IconProps>
) => {
  switch (true) {
    case charging:
      return <Icon type={IconType.ChargingBattery} {...rest} />
    case batteryLevel === 1:
      return <Icon type={IconType.FullBattery} {...rest} />
    case batteryLevel > 0.8:
      return <Icon type={IconType.VeryHighBattery} {...rest} />
    case batteryLevel > 0.6:
      return <Icon type={IconType.HighBattery} {...rest} />
    case batteryLevel > 0.4:
      return <Icon type={IconType.MediumBattery} {...rest} />
    case batteryLevel > 0.2:
      return <Icon type={IconType.LowBattery} {...rest} />
    case batteryLevel > 0:
      return <Icon type={IconType.VeryLowBattery} {...rest} />
    default:
      return <Icon type={IconType.NoBattery} {...rest} />
  }
}

const BatteryIcon: FunctionComponent<BatteryIconProps & IconProps> = ({
  level,
  charging = false,
  height = 2,
  ...rest
}) => {
  return <>{getInteractiveBatteryIcon(level, charging, height, rest)}</>
}

export default BatteryIcon
