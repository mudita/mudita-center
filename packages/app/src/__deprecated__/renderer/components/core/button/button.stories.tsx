/**
 * Copyright (c) Mudita sp. z o.o. All rights reserved.
 * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
 */

import React from "react"
import { storiesOf } from "@storybook/react"
import StoryContainer from "App/__deprecated__/renderer/components/storybook/story-container.component"
import Story from "App/__deprecated__/renderer/components/storybook/story.component"
import {
  DisplayStyle,
  Size,
} from "App/__deprecated__/renderer/components/core/button/button.config"
import Button from "App/__deprecated__/renderer/components/core/button/button.component"
import styled, { css } from "styled-components"
import { IconSize } from "App/__deprecated__/renderer/components/core/icon/icon.component"
import { IconType } from "App/__deprecated__/renderer/components/core/icon/icon-type"

const fullWidthButtonStory = css`
  width: 30rem;
  align-items: stretch;
`

export const FullWidthButton = styled(Button)`
  width: 100%;
`

storiesOf("Components|Core/Button", module)
  .add("Default", () => (
    <>
      <StoryContainer title="Themes">
        <Story title="Primary (default)">
          <Button label="Button" />
        </Story>
        <Story title="Primary disabled">
          <Button disabled label="Button" />
        </Story>
        <Story title="Secondary">
          <Button displayStyle={DisplayStyle.Secondary} label="Button" />
        </Story>
        <Story title="Secondary disabled">
          <Button
            displayStyle={DisplayStyle.Secondary}
            disabled
            label="Button"
          />
        </Story>
      </StoryContainer>
      <StoryContainer title="Sizes">
        <Story title="Big">
          <Button size={Size.FixedBig} label="Button" />
        </Story>
        <Story title="Medium">
          <Button size={Size.FixedMedium} label="Button" />
        </Story>
        <Story title="Small">
          <Button size={Size.FixedSmall} label="Button" />
        </Story>
        <Story title="Custom" customStyle={fullWidthButtonStory}>
          <FullWidthButton label="Button" />
        </Story>
      </StoryContainer>
      <StoryContainer title="Modifiers (icon)">
        <Story title="Primary (default)">
          <Button Icon={IconType.Upload} label="Button" iconSize={IconSize.Small} />
        </Story>
        <Story title="Primary disabled">
          <Button
            Icon={IconType.Upload}
            disabled
            label="Button"
            iconSize={IconSize.Small}
          />
        </Story>
        <Story title="Secondary">
          <Button
            Icon={IconType.Upload}
            displayStyle={DisplayStyle.Secondary}
            label="Button"
            iconSize={IconSize.Small}
          />
        </Story>
        <Story title="Secondary disabled">
          <Button
            Icon={IconType.Upload}
            displayStyle={DisplayStyle.Secondary}
            disabled
            label="Button"
            iconSize={IconSize.Small}
          />
        </Story>
      </StoryContainer>
    </>
  ))
  .add("Link", () => (
    <>
      <StoryContainer title="Link">
        <Story title="Default">
          <Button
            displayStyle={DisplayStyle.Link}
            label="I open Google in new tab"
            href="http://www.google.pl"
          />
        </Story>
        <Story title="With icon">
          <Button
            Icon={IconType.Upload}
            displayStyle={DisplayStyle.Link}
            label="I open Google in new tab"
            href="http://www.google.pl"
            iconSize={IconSize.Medium}
          />
        </Story>
        <Story title="With icon disabled">
          <Button
            disabled
            Icon={IconType.Upload}
            displayStyle={DisplayStyle.Link}
            label="I open Google in new tab"
            href="http://www.google.pl"
            iconSize={IconSize.Medium}
          />
        </Story>
      </StoryContainer>
      <StoryContainer title="Link With Paragraph">
        <Story title="Default">
          <Button
            displayStyle={DisplayStyle.LinkWithParagraph}
            label="I open Google in new tab"
            href="http://www.google.pl"
          />
        </Story>
        <Story title="With icon">
          <Button
            Icon={IconType.Upload}
            displayStyle={DisplayStyle.LinkWithParagraph}
            label="I open Google in new tab"
            href="http://www.google.pl"
            iconSize={IconSize.Small}
          />
        </Story>
      </StoryContainer>
      <StoryContainer title="Action link">
        <Story title="Default">
          <Button
            displayStyle={DisplayStyle.ActionLink}
            label="I open Google in new tab"
            href="http://www.google.pl"
          />
        </Story>
        <Story title="With icon">
          <Button
            Icon={IconType.UploadDark}
            displayStyle={DisplayStyle.ActionLink}
            label="I open Google in new tab"
            href="http://www.google.pl"
            iconSize={IconSize.Small}
          />
        </Story>
      </StoryContainer>
      <StoryContainer title="Menu link">
        <Story title="Default">
          <Button
            nav
            displayStyle={DisplayStyle.MenuLink}
            label="I open Google in new tab"
            href="http://www.google.pl"
          />
        </Story>
        <Story title="With icon">
          <Button
            nav
            Icon={IconType.UploadDark}
            displayStyle={DisplayStyle.MenuLink}
            label="I open Google in new tab"
            href="http://www.google.pl"
            iconSize={IconSize.Bigger}
          />
        </Story>
        <Story title="With icon disabled">
          <Button
            disabled
            nav
            Icon={IconType.UploadDark}
            displayStyle={DisplayStyle.MenuLink}
            label="I open Google in new tab"
            href="http://www.google.pl"
            iconSize={IconSize.Bigger}
          />
        </Story>
      </StoryContainer>
      <StoryContainer title="Tab">
        <Story title="Default">
          <Button
            displayStyle={DisplayStyle.Tab}
            label="I open Google in new tab"
            href="http://www.google.pl"
          />
        </Story>
        <Story title="With icon">
          <Button
            Icon={IconType.UploadDark}
            displayStyle={DisplayStyle.Tab}
            label="I open Google in new tab"
            href="http://www.google.pl"
            iconSize={IconSize.Small}
          />
        </Story>
      </StoryContainer>
      <StoryContainer title="Dropdown">
        <Story title="Default">
          <Button
            displayStyle={DisplayStyle.Dropdown}
            label="I open Google in new tab"
            href="http://www.google.pl"
          />
        </Story>
        <Story title="With icon">
          <Button
            Icon={IconType.Upload}
            displayStyle={DisplayStyle.Dropdown}
            label="I open Google in new tab"
            href="http://www.google.pl"
            iconSize={IconSize.Small}
          />
        </Story>
      </StoryContainer>
    </>
  ))
  .add("Icon", () => (
    <>
      <StoryContainer title="Themes">
        <Story title="Icon only">
          <Button
            Icon={IconType.UploadDark}
            displayStyle={DisplayStyle.IconOnly}
            iconSize={IconSize.Small}
          />
        </Story>
        <Story title="Icon only wiht background">
          <Button
            Icon={IconType.Send}
            displayStyle={DisplayStyle.IconOnlyWithBackground}
            iconSize={IconSize.Big}
          />
        </Story>
        <Story title="Input icon">
          <Button
            Icon={IconType.UploadDark}
            displayStyle={DisplayStyle.InputIcon}
            iconSize={IconSize.Small}
          />
        </Story>
      </StoryContainer>
    </>
  ))
