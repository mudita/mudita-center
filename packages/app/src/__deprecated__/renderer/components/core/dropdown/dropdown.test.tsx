/**
 * Copyright (c) Mudita sp. z o.o. All rights reserved.
 * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
 */

import { fireEvent } from "@testing-library/dom"
import "@testing-library/jest-dom"
import React from "react"
import ButtonComponent from "App/__deprecated__/renderer/components/core/button/button.component"
import { DisplayStyle } from "App/__deprecated__/renderer/components/core/button/button.config"
import Dropdown from "App/__deprecated__/renderer/components/core/dropdown/dropdown.component"
import { renderWithThemeAndIntl } from "App/__deprecated__/renderer/utils/render-with-theme-and-intl"
import Button from "../button/button.component"
import { IconType } from "App/__deprecated__/renderer/components/core/icon/icon-type"

test("matches snapshot", () => {
  const { container } = renderWithThemeAndIntl(
    <Dropdown toggler={<ButtonComponent />} />
  )
  expect(container).toMatchSnapshot()
})

test("renders toggler passed to component", () => {
  const buttonText = "Example"
  const { getByText } = renderWithThemeAndIntl(
    <Dropdown toggler={<ButtonComponent label={buttonText} />} />
  )
  expect(getByText(buttonText)).toBeInTheDocument()
})

test("renders dropdown", () => {
  const buttonText = "Example"
  const { getByTestId, getByText, container } = renderWithThemeAndIntl(
    <Dropdown toggler={<ButtonComponent label={buttonText} />}>
      <Button
        displayStyle={DisplayStyle.Link}
        label="I open Google in new tab"
        href="http://www.google.pl"
        target="_blank"
        Icon={IconType.Upload}
      />
    </Dropdown>
  )

  fireEvent.click(getByText(buttonText))

  expect(getByTestId("dropdown")).toBeVisible()
  expect(container).toMatchSnapshot()
})

test("renders children", () => {
  const buttonText = "Example"
  const childText = "childText"
  const { getByText } = renderWithThemeAndIntl(
    <Dropdown toggler={<ButtonComponent label={buttonText} />}>
      <Button
        displayStyle={DisplayStyle.Link}
        label={childText}
        href="http://www.google.pl"
        target="_blank"
        Icon={IconType.Upload}
      />
    </Dropdown>
  )

  fireEvent.click(getByText(buttonText))

  expect(getByText(childText)).toBeVisible()
})
