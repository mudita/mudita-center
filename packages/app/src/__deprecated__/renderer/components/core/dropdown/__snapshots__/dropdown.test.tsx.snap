// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  background: none;
  padding: 0.1rem 0.7rem;
  cursor: pointer;
  outline: none;
  box-sizing: border-box;
  overflow: hidden;
  -webkit-transition: background 250ms ease-in-out, color 250ms ease-in-out, border 250ms ease-in-out;
  transition: background 250ms ease-in-out, color 250ms ease-in-out, border 250ms ease-in-out;
  width: 17.6rem;
  height: 4rem;
  border-radius: 0.4rem;
  background: #3b3f42;
  border: 0.1rem solid #3b3f42;
}

.c1:hover {
  background: #000000;
}

.c1 p {
  color: #ffffff;
}

.c1 g path {
  fill: #ffffff;
}

.c0 {
  position: relative;
  z-index: 2;
}

.c0 svg path {
  fill: textColor("primary");
}

.c2 {
  position: absolute;
  list-style-type: none;
  padding: 1.4rem 0;
  background-color: #ffffff;
  border-radius: 0.2rem 0.2rem;
  box-shadow: 0 1rem 5.5rem -0.5rem rgba(0,0,0,0.08);
  min-width: 17rem;
  pointer-events: none;
  opacity: 0;
  right: 0;
  top: 100%;
  margin-top: 1rem;
}

<div>
  <div
    class="c0"
  >
    <button
      class="c1"
      size="1"
      type="button"
    />
    <ul
      class="c2"
      data-testid="dropdown"
    />
  </div>
</div>
`;

exports[`renders dropdown 1`] = `
.c2 {
  margin: 0;
  font-size: 1.2rem;
  line-height: 1.67;
  font-weight: 400;
  -webkit-letter-spacing: 0.1rem;
  -moz-letter-spacing: 0.1rem;
  -ms-letter-spacing: 0.1rem;
  letter-spacing: 0.1rem;
  text-transform: uppercase;
  color: #000000;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 2.8rem;
  width: 2.8rem;
  position: relative;
  font-style: normal;
}

.c6 svg {
  width: 100%;
  height: 100%;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  background: none;
  padding: 0.1rem 0.7rem;
  cursor: pointer;
  outline: none;
  box-sizing: border-box;
  overflow: hidden;
  -webkit-transition: background 250ms ease-in-out, color 250ms ease-in-out, border 250ms ease-in-out;
  transition: background 250ms ease-in-out, color 250ms ease-in-out, border 250ms ease-in-out;
  width: 17.6rem;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  height: 2.4rem;
  padding: 0;
  border: none;
  opacity: 0.8;
  width: 100%;
}

.c5 p {
  color: #000000;
}

.c5 g path {
  fill: #000000;
}

.c5:hover {
  opacity: 1;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  background: none;
  padding: 0.1rem 0.7rem;
  cursor: pointer;
  outline: none;
  box-sizing: border-box;
  overflow: hidden;
  -webkit-transition: background 250ms ease-in-out, color 250ms ease-in-out, border 250ms ease-in-out;
  transition: background 250ms ease-in-out, color 250ms ease-in-out, border 250ms ease-in-out;
  width: 17.6rem;
  height: 4rem;
  border-radius: 0.4rem;
  background: #3b3f42;
  border: 0.1rem solid #3b3f42;
}

.c1:hover {
  background: #000000;
}

.c1 p {
  color: #ffffff;
}

.c1 g path {
  fill: #ffffff;
}

.c7 {
  margin-right: 0.4rem;
}

.c3 {
  white-space: nowrap;
}

.c0 {
  position: relative;
  z-index: 3;
}

.c0 svg path {
  fill: textColor("primary");
}

.c4 {
  position: absolute;
  list-style-type: none;
  padding: 1.4rem 0;
  background-color: #ffffff;
  border-radius: 0.2rem 0.2rem;
  box-shadow: 0 1rem 5.5rem -0.5rem rgba(0,0,0,0.08);
  min-width: 17rem;
  pointer-events: auto;
  opacity: 1;
  right: 0;
  top: 100%;
  margin-top: 1rem;
}

<div>
  <div
    class="c0"
  >
    <button
      class="c1"
      size="1"
      type="button"
    >
      <p
        class="c2 c3"
        color="primary"
      >
        Example
      </p>
    </button>
    <ul
      class="c4"
      data-testid="dropdown"
    >
      <a
        class="c5"
        href="http://www.google.pl"
        size="1"
        target="_blank"
      >
        <span
          class="c6 c7"
          data-testid="icon-Upload"
          height="2.8"
          width="2.8"
        >
          <test-file-stub
            image="test-file-stub"
          />
        </span>
        <p
          class="c2 c3"
          color="primary"
        >
          I open Google in new tab
        </p>
      </a>
    </ul>
  </div>
</div>
`;
